### ISSUE TITLE
Support generating vscode.dev deeplinks to file search results

### ISSUE THREAD

**Issue by joyceerhl** [microsoft/vscode-remote-repositories-github#102]
From @eamodio:

> It would be awesome if you could share vscode search links -- like open a vscode.dev link to a specific search
> so I could link someone to a set of search results

**Comment by eamodio** [comment]
Yeah, it would be great to create shareable links for searches to help educate about a codebase. Having the same for reference/implementation search would be cool too (but I know that is harder).
